### ISSUE TITLE
Move background to React component

### ISSUE THREAD

**Issue by extr0py** [extr0py/oni#314]
In `browser\src\UI\index.tsx`, we have this chunk of code:

```
export function setBackgroundColor(backgroundColor: string): void {
    const config = Config.instance()
    const backgroundImageElement: HTMLElement = document.getElementsByClassName("background-image")[0] as HTMLElement
    const backgroundColorElement: HTMLElement = document.getElementsByClassName("background-cover")[0] as HTMLElement
    const backgroundImageUrl = config.getValue<string>("prototype.editor.backgroundImageUrl")
    const backgroundImageSize = config.getValue<string>("prototype.editor.backgroundImageSize") || "cover"

    backgroundImageElement.style.backgroundImage = "url(" + backgroundImageUrl + ")"
    backgroundImageElement.style.backgroundSize = backgroundImageSize
    backgroundColorElement.style.backgroundColor = backgroundColor
    backgroundColorElement.style.opacity = config.getValue<string>("prototype.editor.backgroundOpacity")
}
```

This should be factored out to a React component that is rendered in `RootComponent`, and again driven by state in the redux store.

**Comment by FrigoEU** [comment]
Hey, I'd like to do this, but was wondering what should be in the redux store and what shouldn't. I guess the backgroundColor should be in the store, and the use of the config singleton will stay like that? Or should/will the config object also migrate to the store at some point?

**Comment by extr0py** [comment]
Hi @FrigoEU , sorry for the slow response on my end! Good question about the Redux store. I could see us migrating these config options at some point to the redux store, but I looked at your PR and you did the right thing here - added `backgroundColor` to the redux store and leveraged the rest from config values. I opened #353 to track discussion around this but it isn't pressing and certainly shouldn't block this PR. Overall your PR looks great - thanks for taking this on!
